### ISSUE TITLE
Plex - [list] Users that watch the show/season/episode comparison fails when comparing to a list.

### ISSUE THREAD

**Issue by themarkwilliams** [jorenn92/Maintainerr#1456]
**Describe the bug**
Evaluation of "Using Plex - [list] Users that watch the show/season/episode" equal to a list is incorrect when more than one value is in the second value list.

**To Reproduce**
Rule conditions:
Operator: Or
First Value: Plex -[list] Users that watch show/season/episode
Action: Equals
Second Value: Text
Custom Value: ["user1", "user2"]

**Expected behavior**
Both users should appear in the results from First Value Result. i.e. ["user1","user2"] for it to evaluate to true. Currently, if one appears in the the First Value results, it will evaluate to true.

**Screenshots**
![image](https://github.com/user-attachments/assets/41b49dfe-8c59-42c7-93a2-2b39aa139065)

**Device (please complete the following information):**
 - OS: Ubuntu
 - Version 2.4

**Additional context**
Test media results: 
        - operator: OR
          action: equals
          firstValueName: Plex - [list] Users that watch the show/season/episode
          firstValue:
            - user1
          secondValueName: text
          secondValue:
            - user1
            - user2
          result: true
 
Using Contains (Exact list match) as the action yields the same results.

**Comment by benscobie** [comment]
This is the correct behavior, see the docs: https://docs.maintainerr.info/Rules/#list-examples

If you'd like another operator that does what you describe, raise a feature request here: https://features.maintainerr.info/

**Comment by themarkwilliams** [comment]
Apologies for a follow up, but I'm not tracking. 

If I'm doing an equals comparison, and firstValue doesn't equal secondValue, how is that not a bug?

Why does this example in your list documentation evaluate to true when first value is not an exact match of secondValue?
![image](https://github.com/user-attachments/assets/9cf4455a-5689-4592-9bd6-4a0d1ec12320)

If the operator were equals in that example, the result should be false. 

Or what am I missing?

**Comment by ydkmlt84** [comment]
If you want to come on over to the Discord I can try to help you understand.

**Comment by benscobie** [comment]
> Or what am I missing?

It's an exact string match of ANY item in your list, not all of the provided list values. AND vs OR. The order does not matter.

When we add the logic you want we should come up with some better descriptions for sure.

**Comment by themarkwilliams** [comment]
Got it. Any, not All for the Contains (Exact).  

Should there be a behavior difference for Equals vs Contains (Exact)? 

I'll try the additional, separate rule as a workaround for the multiple checks. 

The All comparison would be very nice to have and I agree there is an opportunity to clarify the descriptions. 

Thanks for the help!

**Comment by benscobie** [comment]
> Should there be a behavior difference for Equals vs Contains (Exact)?

For arrays, equals checks that every element in firstValue is in the secondValue array (case insensitive). It's not a perfect equals, as the secondValue array could contain a value not within firstValue and it would pass.

**Comment by ydkmlt84** [comment]
Equals will be fixed once #1489 is merged.
